src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -419,6 +426,10 @@ def register_modules(self, **kwargs):
             if module is None:
                 register_dict = {name: (None, None)}
             else:
+                # register the original module, not the dynamo compiled one
+                if "_dynamo.eval_frame" in str(module.__class__):
```

Comment:
Think this check is better:
```py
if is_torch_version(">=", "2.0.0") and isinstance(sub_model, torch._dynamo.eval_frame.OptimizedModule):
```

Reply:
As commented above, `torch._dynamo` does not exist in all platforms.

Reply:
Ah sorry, now I get it